news/gis_agents.rst

```diff
@@ -0,0 +1,17 @@
+**Added:** None
```

Comment:
Please remove the `None` in the section where you have added entries

Reply:
Thank you.
I've removed them.